AI/VCAI/HeroManager.h

```diff
@@ -0,0 +1,107 @@
+/*
+* HeroManager.h, part of VCMI engine
+*
+* Authors: listed in file AUTHORS in main folder
+*
+* License: GNU General Public License v2.0 or later
+* Full text of license available in license.txt file, in main folder
+*
+*/
+
+#pragma once
+
+#include "AIUtility.h"
+
+#include "../../lib/GameConstants.h"
+#include "../../lib/VCMI_Lib.h"
+#include "../../lib/CTownHandler.h"
+#include "../../lib/CBuildingHandler.h"
+#include "VCAI.h"
+
+enum HeroRole
+{
+	MAIN,
+
+	SCOUT
+};
+
+class DLL_EXPORT IHeroManager //: public: IAbstractManager
```

Comment:
If you export something you should import it somewhere, in case of AI you export classes only to be used in tests.

Reply:
Yeah, one day I may decide to write some tests. Now this can be removed.

Reply:
But it can not be just removed, `IHeroManager` is a base class of AIHelper that is already exported.